src/org/opensolaris/opengrok/index/Indexer.java

```diff
@@ -263,7 +264,12 @@ public static void main(String argv[]) {
                 Message m = Message.createMessage("config");
                 m.addTag("set");
                 m.setText("projectsEnabled = true");
+                try {
                 m.write(host, port);
+                } catch (ConnectException ce) {
```

Comment:
I'm not quite clear why this exception is now being swallowed in a Lucene upgrade.

Reply:
same as above, error connecting to webapp shouldn't block indexing